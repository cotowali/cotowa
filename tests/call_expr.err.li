fn f(a int) {

}

unknown_fn()

var v = 0
v()

f(0)
f("a")
f()
f(0, 1)
