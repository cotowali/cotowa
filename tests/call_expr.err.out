call_expr.err.li 5,1: function `unknown_fn` is not defined
    5 | unknown_fn()
      | ^^^^^^^^^^^^
call_expr.err.li 8,1: `v` is not function (`int`)
    8 | v()
      | ^^^
call_expr.err.li 11,3: mismatched types: `int` (expected) and `string` (found)
   11 | f("a")
      |   ^^^
call_expr.err.li 12,1: expected 1 arguments, but got 0
   12 | f()
      | ^^^
call_expr.err.li 13,1: expected 1 arguments, but got 2
   13 | f(0, 1)
      | ^^^^^^^
